locales/sv_SE/pcsx2_Iconized.po

```diff
@@ -623,33 +641,33 @@ msgstr ""
 "KonfigurationsDialogen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:29
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "50% cyclerate. Significant reduction of CPU requirements. Speedup for very "
 "lightweight games, slows down others. FMVs and audio may stutter or skip."
 msgstr ""
-"50% Cykelgrad. Betydlig minskning av CPU-krav.\n"
+"50% cykelgrad. Betydlig minskning av CPU-krav.\n"
```

Comment:
In this context 'grad' makes little sense, 'cykeltakt' is better here.

Reply:
Agreed - I'll change it.